public/js/reducers/user.js

```diff
@@ -24,5 +24,13 @@ export default function user(state, action) {
     });
   }
 
+  if (action.type === actionTypes.RX_BRAINTREE_TOKEN) {
+    // As we want to stay signed in if we are
+    // this reducer inherits state.
+    return Object.assign({}, defaults, state, {
+      braintreeToken: action.user.braintreeToken,
```

Comment:
per previous comments, I think it's nice to keep this data specific to the action and just call it `action.braintree_token`